src/components/CircularProgressBar/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from "react";
+import {
+  CircularProgressbarWithChildren,
+  buildStyles,
+} from "react-circular-progressbar";
+import { CircularProgressbarProps } from "react-circular-progressbar/dist/types";
+import { ReactNode } from "react-transition-group/node_modules/@types/react";
+import colors from "../../style/colors";
+
+export interface CircularProgressBarPropType
+  extends Partial<CircularProgressbarProps> {
+  percentage?: number;
+  children?: ReactNode;
+}
+
+export const CircularProgressBar: FC<CircularProgressBarPropType> = ({
+  percentage = 66,
```

Comment:
Why default to 66?

Reply:
True, I though I had changed that to 0. Will change! Thx